tests/wpt/metadata/html/semantics/embedded-content/media-elements/playing-the-media-resource/play-in-detached-document.html.ini

```diff
@@ -1,6 +0,0 @@
-[play-in-detached-document.html]
```

Comment:
IIUC this test should continue timing out.

Reply:
Yes, after fixing the test with `step_func`, it gets back to `TIMEOUT` and it's more reasonable! Thanks!